diagnostics/app/views/feedback.scala.html

```diff
@@ -0,0 +1,36 @@
+@(page: model.Page, emailParams: String, uri: Option[String])(implicit request: RequestHeader)
+
+@main(page){ }{
+<div class="l-side-margins">
```

Comment:
Are you sure this page won't rot as the site style updates, would it be possible to do it in Composer with a code embed or something which would future proof it a bit more?